wherescape/connectors/hubspot/python_hubspot_check_missing_columns.py

```diff
@@ -0,0 +1,153 @@
+    """
+    sql = "SELECT lt_obj_key FROM ws_load_tab WHERE lt_table_name = ?"
+    results = wherescape_instance.query_meta(sql, [table_name])
+    table_rows = []
+
+    if len(results) > 0:
+        lc_obj_key = results[0][0]
+        sql = "SELECT lc_col_name FROM ws_load_col WHERE lc_obj_key = ?"
+        results = wherescape_instance.query_meta(sql, [lc_obj_key])
+        wherescape_columns = [item[0] for item in results]
+        api_response = None
+        try:
+            api_response = hubspot_instance.client.crm.properties.core_api.get_all(
+                object_type=object_type, archived=False
+            )
+        except ForbiddenException:
+            logging.info(
+                f"No access for type {object_type} via the Hubspot api for {environment}"  # noqa: E501
+            )
+        else:
+            api_results = api_response.to_dict()
+            hubspot_columns = []
+            for result in api_results["results"]:
+                hubspot_columns.append(result["name"])
+            missing_in_wherescape, missing_in_hubspot = compare_columns(
+                wherescape_columns, hubspot_columns
+            )
+            table_rows = create_table_rows(
+                missing_in_wherescape,
+                missing_in_hubspot,
+                environment,
+                table_name,
+            )
+    else:
+        logging.info(
+            f"No load table exists for Hubspot object {object_type} and environment {environment} ({table_name})"  # noqa: E501
+        )
+
+    return table_rows
+
+
+def compare_load_and_ds(wherescape_instance, table_rows):
+    """
+    Function that compares the ds table rows with the supplied table_rows. It
+    will push the diffence to the load table to be load table which will be
+    added to the ds table later in the process.
+    """
+    # Get the datastore rows to compare to the new load rows
+    ds_rows = get_ds_rows(wherescape_instance)
+    # For comparison we need tuples instead of lists
+    ds_rows = [tuple(row) for row in ds_rows]
+    new_rows_in_load_table = [row for row in table_rows if row not in ds_rows]
+    if len(new_rows_in_load_table) > 0:
+        for row in new_rows_in_load_table:
+            logging.warning(
+                f"Hubspot missing column: the {row[0]} environment in {row[3]}: table {row[2]} is missing column {row[1]}"  # noqa: E501
+            )
+    else:
+        logging.info("No new missing rows in Hubspot objects found.")
+
+    # Pushing the missing columns to the load table
+    wherescape_instance.push_many_to_target(
+        "insert into %s values (?, ?, ?, ?, null, null)"
+        % wherescape_instance.load_full_name,
+        new_rows_in_load_table,
+    )
+
+
+def create_metadata(wherescape_instance):
+    """
+    Function to create the metadata columns for the missing Hubspot columns
+    check.
+    """
+    sql = prepare_metadata_query(
+        lt_obj_key=wherescape_instance.object_key,
```

Comment:
Your now leaving the naming of the load table up to user of the function. I think I would have hardcoded it